using_images/s2i_images/dot_net_core.adoc

```diff
@@ -111,55 +109,63 @@ available to the build steps.
+*_csproj_* or *_fsproj_*) or a folder containing a single project file.
 |*_._*
 
-|`DOTNET_PUBLISH`
-|Used to control whether the application should be built by executing `dotnet
-build` or `dotnet publish`. To publish the application, set the value to `true`.
-It is recommended to publish your application.
-|For backwards compatibility, the
-default is `false`.
-
-In the next major release, this variable will be removed and the builder will
-always publish the application.
+|`DOTNET_SDK_VERSION`
+|Used to select the default SDK version when building. If there is a
+*_global.json_* file in the source repository, that takes precedence. When set
+to `latest`, the latest SDK in the image is used.
+|Lowest SDK version available in the image.
 
 |`DOTNET_ASSEMBLY_NAME`
 |Used to select the assembly to run. This must _not_ include the `.dll`
-extension. Set this to the output assembly name specified in *_project.json_*
-(`name`, `buildOptions/outputName`). For *_project.json_*, the assembly name
-defaults to the *_project.json_* parent folder. When *_project.json_* is at the
-*_context-dir_*, the parent folder name will be *_src_*. So, by default, this
-generates a `src.dll` assembly. Setting `DOTNET_ASSEMBLY_NAME` will cause:
-
-  - The assembly to be `<DOTNET_ASSEMBLY_NAME>.dll`
-  - The application sources to be in subfolder `DOTNET_ASSEMBLY_NAME` in the deployed container.
-
-|The name of the `DOTNET_STARTUP_PROJECT` folder.
+extension. Set this to the output assembly name specified in *_csproj_* (`PropertyGroup`/`AssemblyName`).
+|The name of the *_csproj_* file.
 
 |`DOTNET_RESTORE_SOURCES`
 |Used to specify the space-separated list of *NuGet* package sources used during
 the restore operation. This overrides all of the sources specified in the
 *_NuGet.config_* file.
-|Unset
+|
+
+|`DOTNET_TOOLS`
+|Used to specify a list of .NET tools to install before building the application.
+To install a specific version, add `@<version>` to the end of the package name.
+|
 
 |`DOTNET_NPM_TOOLS`
 |Used to specify a list of NPM packages to install before building the application.
-|Unset
+|
 
 |`DOTNET_TEST_PROJECTS`
-|Used to specify the space-separated list of test projects to run. This must be folders containing
- *_project.json_*. `dotnet test` is invoked for each folder.
-|Unset
+|Used to specify the list of test projects to test. This must be project files or folders containing a single project file. `dotnet test` is invoked for each item.
+|
 
 |`DOTNET_CONFIGURATION`
 |Used to run the application in `Debug` or `Release` mode. This value should be
 either `Release` or `Debug`.
 |`Release`
 
+|`DOTNET_VERBOSITY`
+|Used to specify the verbosity of the dotnet build commands. When set, the
+environment variables are printed at the start of the build. This variable can
+be set to one of the msbuild verbosity values (`q`[uiet], `m`[inimal], `n`[ormal],
+`d`[etailed], and `diag`[nostic]).
```

Comment:
nit: the long names are also accepted by the tooling, so we should include them in the backticks:
`q[uiet]`, `m[inimal]`, `n[ormal]`,
+`d[etailed]`, and `diag[nostic]`